Homework_9/task_1.py

```diff
@@ -0,0 +1,95 @@
+# Напишите следующие функции:
+# * Нахождение корней квадратного уравнения
+# * Генерация csv файла с тремя случайными числами в каждой строке. 100-1000 строк.
+# * Декоратор, запускающий функцию нахождения корней квадратного уравнения с каждой тройкой чисел из csv файла.
+# * Декоратор, сохраняющий переданные параметры и результаты работы функции в json файл.
+
+import csv
+import random
+import json
+from functools import wraps
+
+def quadratic_equation(a, b, c):
+    """
+    Функция для нахождения корней квадратного уравнения.
+    Возвращает корни уравнения в виде списка.
+    """
+    # Находим дискриминант
+    discriminant = b**2 - 4*a*c
+    
+    # Проверяем условия дискриминанта
+    if discriminant > 0:
+        x1 = (-b + discriminant**0.5) / (2*a)
+        x2 = (-b - discriminant**0.5) / (2*a)
```

Comment:
Пожалуйста, не забывайте про PEP8.
```suggestion
        x1 = (-b + discriminant ** 0.5) / (2 * a)
        x2 = (-b - discriminant**0.5) / 2 / a
```